examples/gcp-passthrough-nlb/main.tf

```diff
@@ -0,0 +1,39 @@
+
+
+data "google_client_config" "default" {}
+
+provider "google" {}
+
+data "google_compute_instance" "ghe" {
+  project = var.project_name
+  zone    = var.zone
+  name    = var.instance_name
+}
+
+data "google_compute_subnetwork" "ghe" {
+  project = var.project_name
+  name    = var.subnet_name
+  region  = var.region
+}
+
+module "sourcegraph-psc" {
+  source = "../../modules/gcp-passthrough-nlb"
+
+  project_id = "sourcegraph-github-enterprise"
+  network_id = "ghe"
```

Comment:
why not also use a data source here? for consistency sake

Reply:
Nice catch, updated